hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/calcite/opt/physical/index/PhysicalIndexExpressionTest.java

```diff
@@ -189,4 +190,9 @@ private void checkNoIndexForCondition(String condition, String hdRemainderFilter
             checkNoIndex(sql, INT, INT, INT, INT, INT);
         }
     }
+
+    @RunWith(HazelcastSerialClassRunner.class)
+    @Category({QuickTest.class, ParallelJVMTest.class})
+    public static class PhysicalSortIndexTest extends OptimizerTestSupport {
```

Comment:
Looks like this is a leftover.

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/18013/commits/58a23683393025c09b7ccceb41d32c3105917ae4